python/hail/java.py

```diff
@@ -1,21 +1,50 @@
-
-def jarray(gateway, jtype, lst):
-    jarr = gateway.new_array(jtype, len(lst))
+def jarray(jtype, lst):
+    jarr = Env.gateway().new_array(jtype, len(lst))
     for i, s in enumerate(lst):
         jarr[i] = s
     return jarr
 
+
 def scala_object(jpackage, name):
     return getattr(getattr(jpackage, name + '$'), 'MODULE$')
 
+
```

Comment:
module-level functions are treated like classes according to my intelliJ python-style.  I've mostly just been autoformatting as I go, and assuming that's what we want.